fava/api/__init__.py

```diff
@@ -90,6 +90,19 @@ def _sidebar_links(entries):
             for entry in sidebar_link_entries]
 
 
+def _settings(entries):
```

Comment:
This function should also parse and validate the options (and generate beancount-errors if they're incorrectly specified)